README.md

```diff
@@ -1,2 +1,34 @@
 # overops-functions
 Public OverOps User Defined Functions
+
+Full manifest of all libraries can be found here - https://git.io/fxDIW
+
+## Our Collection consists of two types of functions:
+### Anomaly Functions:
+An Anomaly UDF is a function that would be used to determine whether the volume of specific events is considered anomalous by comparing it to a target threshold within a view/timeframe.
```

Comment:
i would add that this is a periodic check and that it can be adjusted to 5/15/60... minutes or something

Reply:
added